pennylane/ops/op_math/exp.py

```diff
@@ -32,12 +32,13 @@
     OperatorPropertyUndefined,
     Tensor,
 )
-from pennylane.ops.qubit import Hamiltonian
 from pennylane.wires import Wires
 
 from .sprod import SProd
 from .sum import Sum
+from .linear_combination import LinearCombination
 from .symbolicop import ScalarSymbolicOp
+from ..qubit.hamiltonian import Hamiltonian
```

Comment:
Why is `Hamiltonian` imported from this path?

Reply:
Otherwise I got cyclic import errors